crates/chainio/clients/avsregistry/src/writer.rs

```diff
@@ -330,19 +333,190 @@ impl AvsRegistryChainWriter {
         info!(tx_hash = ?tx,"succesfully deregistered operator with the AVS's registry coordinator" );
         Ok(*tx.tx_hash())
     }
+
+    /// Sets minimum stake for the quorum
+    ///
+    /// Can only be caled by registry coordinator's owner.
+    /// The quorum numbers must exist, else the tx will fail.
+    /// # Arguments
+    ///
+    /// # `quorum_numbers` - The respective quorum numbers in [`u8`].
+    /// # `minimum_stake` - The minimum stake in [`U96`].
+    ///
+    /// # Returns
+    ///
+    /// * [`TxHash`] - The transaction hash of the transaction.
+    pub async fn set_minimum_stake_for_quorum(
+        &self,
+        quorum_numbers: u8,
```

Comment:
This is for a single quorum, right? Then we should rename this to `quorum_number`. Also, let's use the `QuorumNum` type instead of `u8`.